metldata/model_utils/manipulate.py

```diff
@@ -47,6 +52,19 @@ def __init__(self, class_name: str, slot_name: str):
         super().__init__(message)
 
 
+def get_normalized_slot_usage(*, class_: ClassDefinition) -> dict[str, SlotDefinition]:
+    """Get a normalized slot usage dictionary from a class definition."""
+
+    if class_.slot_usage:
+        if isinstance(class_.slot_usage, jsonasobj2.JsonObj):
+            return jsonasobj2.as_dict(class_.slot_usage)
```

Comment:
```suggestion
        if hasattr(class_.slot_usage, "_as_dict"):
            return class_.slot_usage._as_dict
```

Then we wouldn't need jsonasobj2.